front_end/Matching/MatchingBrowseThree.js

```diff
@@ -0,0 +1,28 @@
+import React, {Component} from 'react';
+import {SafeAreaView, View, Text, TouchableOpacity} from 'react-native';
+
+import styles from '../Static/style.js';
+
+export default class BrowseThreeScreen extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <SafeAreaView style={styles.match__background}>
+        <View style={styles.screen__header} />
+        <View style={[styles.content__container, styles.content__centering]}>
+          <Text style={[{fontWeight: 'bold', fontSize: 20, margin: 10}]}>
```

Comment:
Why is this a hardcoded style instead of drawing from the styles file?

Reply:
Updated